docs/version_releases.rst

```diff
@@ -0,0 +1,20 @@
+Version releases
```

Comment:
What do you think if we move that information to the Falcon Development section?

Reply:
Done.